search-enterprise/latest/support/index.md

```diff
@@ -11,7 +11,8 @@ The following are the supported platforms for Search Enterprise 3.1.x:
 | Content Services 7.2.x | Including Alfresco ActiveMQ 5.15.8, Alfresco Transform Service 1.4.1 and PostgreSQL 11.7 |
 | Content Services 7.1.x | Including Alfresco ActiveMQ 5.15.8, Alfresco Transform Service 1.4.1 and PostgreSQL 11.7 |
 | JDK 11 or OpenJDK 11 | |
-| Elasticsearch server 7.10.1 | |
-| Alfresco Elasticsearch Connector 3.1.x | |
+| Elasticsearch server 7.17.3 | |
```

Comment:
This should state Elasticsearch versions 7.10 - 7.17 inclusive. Not just 7.17

Reply:
This has been added to PR https://github.com/Alfresco/docs-alfresco/pull/839

Reply:
Thanks @mjstanford, yes that text is covered in the PR I mention. I am referring to this JIRA https://alfresco.atlassian.net/browse/DOCS-6945